### PR TITLE
Improve French translation

### PR DESCRIPTION
**Pull request by reda-alaoui**

### CODE REVIEW

**Comment by jonkoops** [comment]
@reda-alaoui could you get another French native speaker to review your PR?

**Comment by reda-alaoui** [comment]
@jonkoops One of my colleague can do that if you want. I don't where to find one otherwise.

**Comment by jonkoops** [comment]
That is fine, it always good to have another set of eyes on this. If your colleague can review it that would be great!

File: themes/src/main/resources-community/theme/base/login/messages/messages_fr.properties [hunk 1]
```diff
@@ -386,6 +386,35 @@ console-email-code=Code courriel:
...
+auth-username-form-display-name=Nom d''utilisateur
+auth-username-form-help-text=Commencez la connexion en saisissant votre nom d''utilisateur.
+auth-username-password-form-display-name=Nom d''utilisateur et mot de passe
+auth-username-password-form-help-text=Connectez-vous en saisissant votre nom d''utilisateur et votre mot de passe.
+
+# WebAuthn
+webauthn-display-name=Cl\u00e9 de S\u00e9curit\u00e9
+webauthn-help-text=Utilisez votre cl\u00e9 de s\u00e9curit\u00e9 pour vous connecter.
+webauthn-passwordless-display-name=Cl\u00e9 de S\u00e9curit\u00e9
+webauthn-passwordless-help-text=Utilisez votre cl\u00e9 de s\u00e9curit\u00e9 pour vous connecter sans mots de passe.
```
Comment (roman-garcia) [inline themes/src/main/resources-community/theme/base/login/messages/messages_fr.properties:401]: sans mots de passe => sans mot de passe

File: themes/src/main/resources-community/theme/base/login/messages/messages_fr.properties [hunk 2]
```diff
@@ -386,6 +386,35 @@ console-email-code=Code courriel:
...
+webauthn-unsupported-browser-text=WebAuthn n''est pas support\u00e9 par ce navigateur. Essayez une autre m\u00e9thode ou contactez votre administrateur.
+webauthn-doAuthenticate=Se connecter avec une Cl\u00e9 de S\u00e9curit\u00e9
+webauthn-createdAt-label=Cr\u00e9\u00e9 le
+
+# WebAuthn Error
+webauthn-error-title=Erreur lors de l''utilisation de la Cl\u00e9 de S\u00e9curit\u00e9
+webauthn-error-registration=L''enregistrement de la Cl\u00e9 de S\u00e9curit\u00e9 a \u00e9chou\u00e9.<br/> {0}
+webauthn-error-api-get=L''authentification via la Cl\u00e9 de S\u00e9curit\u00e9 a \u00e9chou\u00e9.<br/> {0}
+webauthn-error-different-user=Le premier utilisateur authentifi\u00e9 ne correspond pas \u00e0 celui qui est authentifi\u00e9 par la Cl\u00e9 de S\u00e9curit\u00e9.
+webauthn-error-auth-verification=Le r\u00e9sultat de l''authentification produit par la cl\u00e9 de s\u00e9curit\u00e9 est invalide.<br/> {0}
```
Comment (roman-garcia) [inline themes/src/main/resources-community/theme/base/login/messages/messages_fr.properties:414]: l''authentication produit => l''authentication produite

**Review by roman-garcia — CHANGES_REQUESTED** [review]

**Review by roman-garcia — APPROVED** [review]

**Review by jonkoops — APPROVED** [review]

**Comment by reda-alaoui** [comment]
@jonkoops are we waiting for another reviewer?

**Comment by jonkoops** [comment]
Yes, we need a maintainer to approve this. I'll see if one is available to do so.

**Review by ssilvert — APPROVED** [review]

**Pull request merged**